src/Helpers/CacheMap.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Backpack\Basset\Helpers;
+
+use Illuminate\Support\Facades\File;
+use Illuminate\Support\Facades\Storage;
+
+class CacheMap
+{
+    private $map = [];
+    private $path;
+    private $active = false;
+    private $dirty = false;
```

Comment:
Maybe we add a verb here to make the call more fluent? I think I like `cachemap->isDirty` more than `cachemap->dirty()`, how about you?

```suggestion
    private $isActive = false;
    private $isDirty = false;
```